docs/_guide/start.md

```diff
@@ -8,131 +8,312 @@ slug: start
+### Add an element
 
-```html
-<head>
-  <script type="module" src="node_modules/package-name/existing-element.js"></script>
-</head>
-<body>
-  <existing-element></existing-element>
-</body>
-```
+Pick a location to store your LitElement components, and create a test element: 
 
-To import into another JavaScript module, use the component's package name:
+_components/my-element.js_
 
 ```js
-import 'package-name/existing-element.js';
+import {LitElement, html} from 'lit-element';
 
-class MyElement extends LitElement{
-  render(){
+class MyElement extends LitElement {
+  render() {
     return html`
-      <existing-element></existing-element>
+      <div>Hello from MyElement!</div>
     `;
   }
 }
+
 customElements.define('my-element', MyElement);
 ```
+
+Import your component:
+
+Index.js
+
+```js
+import './components/my-elements.js';
+```
+
+Use your component:
+
+index.html
+
+```html
+<my-element></my-element>
+```
+
+At this point, you should be able to build and run your project and see the "Hello from MyElement!" message.
+
+<div class="alert alert-info">
+
+**Adjust for your build system.** How you import the component may vary slightly depending on your build system and project structure. 
+
+</div>
+
+
+### Optional: Use ES dev server
+
+If you already have a dev server that works with your build system, it should work with LitElement. [ES dev server](https://open-wc.org/developing/es-dev-server.html) is an alternative dev server that provides a simple, bundler-free workflow. It performing a small number of useful transforms, including transforming bare module specifiers, transpiling to ES5 when needed, and adding polyfills for older browsers.
+
+1. Install es-dev-server
+
+    ```bash
+    npm i -D es-dev-server
+    ```
+
+1. Add a dev server command to `package.json`: 
+
+
+    ```json
+    {
+      "scripts": {
+        "start": "es-dev-server --app-index index.html --node-resolve --watch --open"
+      }
+    }
+    ```
+
+1. Start ES dev server:
+
+    ```bash
+    npm run start
+    ```
+
+### Supporting older browsers
+
+To support older browsers that don't support ES6 and the web components specifications, you'll need to take a few extra steps:
+
+*   Compile to ES5. Your build will also need to transform JavaScript modules into a format compatible with older browsers, like AMD. 
+*   Include the [web components polyfills](https://github.com/webcomponents/polyfills/tree/master/packages/webcomponentsjs). In almost all cases, you'll also want to include [custom-elements-es5-adapter.js](https://github.com/webcomponents/polyfills/tree/master/packages/webcomponentsjs#custom-elements-es5-adapterjs) to ensure that your ES5 version can also run on newer browsers. 
+
+If you need to support older browsers, there are two basic approaches you can take:
+
+*   Serve the ES5 version of the project to all browsers.
+*   Build both ES6 and ES5 versions of your project, and serve the ES6 version to modern browsers only. 
```

Comment:
I would mention [webpack-babel-multi-target-plugin](https://github.com/DanielSchaffer/webpack-babel-multi-target-plugin) which allows to build both ES6 and ES5 versions. 
At Vaadin, we have been successfully using it to ship ES5 code to IE11 and Safari 9.

Reply:
I think we should probably get into those details in the tools section, not getting started.